src/SLIP10Node.ts

```diff
@@ -152,6 +153,10 @@ export class SLIP10Node implements SLIP10NodeInterface {
     return bufferToBase64String(this.keyBuffer);
   }
 
+  get #privateKeyBuffer(): Buffer {
+    return this.keyBuffer.slice(0, 32);
```

Comment:
Can we leave a comment to explain why this is needed at this point? Even if we are refactoring it in a later PR. As discussed on Slack.